homeassistant/components/n26/sensor.py

```diff
@@ -30,15 +30,20 @@
 def setup_platform(
         hass, config, add_entities, discovery_info=None):
     """Set up the N26 sensor platform."""
-    api_data = hass.data[DOMAIN][DATA]
+    api_list = hass.data[DOMAIN][DATA]
```

Comment:
Please add a guard clause here that checks if `discovery_info` is None and return if so.